src/platform/packages/shared/shared-ux/chrome/navigation/src/ui/components/panel/panel_group.tsx

```diff
@@ -83,18 +83,21 @@ export const PanelGroup: FC<Props> = ({ navNode, isFirstInList, hasHorizontalRul
     }
   }
 
-  const renderChildren = useCallback(() => {
-    if (!filteredChildren) return null;
+  const renderChildren = useCallback(
+    (parentIsAccordion?: boolean) => {
```

Comment:
nit:
```suggestion
    ({ parentIsAccordion } = { parentIsAccordion: false }) => {
```

and change the call on line 122 to:
```tsx
            {renderChildren({ parentIsAccordion: true })}
```